examples/hello-electron/README.md

```diff
@@ -20,5 +20,5 @@ And edit your env file with the proper values.
 To run the example app, from the root of the project, run:
 
 ```bash
-npm run start:electron-example
+nx serve hello-electron
```

Comment:
The one downside with this is people need to have the right nx version globally

Reply:
Fair. I guess my preference is not to have to maintain an npm script pointing to an nx command. I can just update the readme to `npx nx serve hello-electron`.

Reply:
https://github.com/bitdriftlabs/capture-es/pull/41/files